imports/startup/server/index.coffee

```diff
@@ -168,6 +168,43 @@ Meteor.methods
       """
     makeRequest(query)
 
+  'getTrendingInfectiousAgents': (date, date2, days) ->
+    query = prefixes + """
+      SELECT ?resolvedTerm
+        (sample(?termLabel) as ?word)
+		    (sample(?termLabel2) as ?word2)
+        (count(DISTINCT ?article) as ?count)
+		    (sample(?c2) as ?count2)
+        ((?count2)/(xsd:float(#{escape(days)}*4)) as ?rate2)
+        ((?count)/(xsd:float(#{escape(days)})) as ?rate)
+  		  ((xsd:float(?rate)/(xsd:float(?rate2))) AS ?result)
```

Comment:
It seems like the rates should already be floats. Are you sure we need to recast them?